modules/text2vec-cohere/clients/cohere.go

```diff
@@ -172,6 +179,48 @@ func (v *vectorizer) getCohereUrl(ctx context.Context, baseURL string) string {
 	return v.urlBuilder.url(passedBaseURL)
 }
 
+func (v *vectorizer) GetApiKeyHash(ctx context.Context, config moduletools.ClassConfig) [32]byte {
```

Comment:
what is a purpose of this method?

Reply:
Different keys have different rate limits. Now we do not want to check the rate limits again at the start of each batch so we need to find a way to detect if the API key has changed (and we need to recheck the rate limit). Now I do not want to keep the API key longer in memory than necessary and therefore use the hash of it